src/rastervision/tagging/data/planet_kaggle.py

```diff
@@ -282,10 +282,9 @@ def make_batch(self, img_batch, file_inds):
 class PlanetKaggleTiffFileGenerator(PlanetKaggleFileGenerator):
     def __init__(self, datasets_path, options):
         self.dev_dir = 'train-tif-v2'
-        self.test_dir = 'test-tif-v2'
-        self.file_names = [
-            'train-tif-v2.zip', 'test-tif-v2.zip', 'train_v2.csv.zip',
-            'planet_kaggle_tiff_channel_stats.json']
+        self.test_dir = 'test-tif-v3'
+        self.zip_file_names = [
```

Comment:
When resolving merge conflicts, you mistakenly changed `file_names` to `zip_file_names` and also omitted the channel stats file.